source/articles/get-started/asp-dotnet-core/index.md

```diff
@@ -0,0 +1,215 @@
+
+> [!TIP]  
+>a public constructor that accepts `LinqToDbConnectionOptions<AppDataConnection>` and passes the options on to the base constructor is required.
+
+## Add Connection String
+
+For this example we're going to use SQLite in memeory mode, for production you'll want to use something else, but it's pretty easy to change.
+
+First you want to add the connection string to `appsettings.Development.json`, something like this: 
+```json
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  //add this
+  "ConnectionStrings": {
+    "Default": ":memory:" //<-- connection string, used in the next step
+  }
+}
+```
+
+## Configure Dependency injection
+
+inside `Startup.cs` you want register the data connection like this:
+```C#
+public class Startup
+{
+    public IConfiguration Configuration { get; }
+
+    public void ConfigureServices(IServiceCollection services)
+    {
+        //...
+        //using LinqToDB.AspNet
+        services.AddLinqToDbContext<AppDataConnection>((provider, options) => {
+            options
+            //will configure the AppDataConnection to use
+            //sqite with the provided connection string
+            //there are methods for each supported database
+            .UseSQLite(Configuration.GetConnectionString("Default"))
+            //default logging will log everything using the ILoggerFactory configured in the provider
+            .UseDefaultLogging(provider);
+        });
+        //...
+    }
+}
+```
+
+> [!TIP]  
+> There's plenty of other configuration options availble, if you are familier with LINQ To DB alreadyyou can convert your existing application over to use the new `LinqToDbConnectionOptions` class as every configuration method is supported
+
+> [!TIP]  
+> Note, only `DataConnection` supports `LinqToDbConnectionOptions`. `DataContext` is not yet supported.
+
+> [!TIP]  
+> Use `AddLinqToDbContext<TContext, TContextImplementation>` if you would like to resolve an interface or baseclass instead of the concrete class in your controllers
+
+By Default this will configure the service provider to create a new `AppDataConnection` for each HTTP Request, and will dispose of it once the request is finished. This can be configured with the last parameter to `AddLinqToDbContext(... ServiceLifetime lifetime)`, more information about lifetimes [here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-3.1#service-lifetimes)
+
+## Simple Entity Configuration
+
+Let's create this simple entity in our project
+
+```C#
+public class Person
+{
+    public Guid Id { get; set; } = Guid.NewGuid();
+    public string Name { get; set; }
+    public DateTime Birthday { get; set; }
+}
+```
+
+By default the Id column will be identitifed as the primary key.
```

Comment:
We don't have by-name conventions like EF.core, so probably it is not true

Reply:
Thanks, I added an attribute and remove the note.